src/widgetutility.cpp

```diff
@@ -25,6 +25,14 @@ void onHeaderContextMenu(QTreeView* view, const QPoint& pos)
     checkBox->setText(columnName);
     checkBox->setChecked(!header->isSectionHidden(i));
 
+    // Enable the checkbox if 1) the section is visible, or 2) the
```

Comment:
The `setChecked()` above is now redundant.

Reply:
Are you sure? Enabled != Checked.

Reply:
Bah.